lib/elixir_console_web/live/console_live/history_component.html.leex

```diff
@@ -1,5 +1,9 @@
 <div class="flex-1"></div>
 <div class="p-2" id="commandOutput" phx-update="append">
+  <div id="version-info">
```

Comment:
I think this block would look more integrated with the rest of the commands if we used the same classes used in the command divs

Reply:
Didn't get what you mean. Which classes are you referring to?

Reply:
`text-gray-300 font-medium`
Right now it is hard to read because the text is black over a dark green background

Reply:
Thats something it told Gonchi, by default it was black and when I saw it I thought that it was looking good as it is differentiated from the commands. What do you think?

Reply:
That's good to differentiate it from the commands. How about using text-gray-500 or 600? Those look different and are still readable, darker than that I find it hard to read.
`font-medium` makes all font look with the same weight, so I prefer to have it there also so it doesn't look like we accidentally forgot to make this part bold

Reply:
Thats good

Reply:
Gray feels like a good contrast with respect to background while still being different from the evaluation output :+1: